src/machine/i2c.go

```diff
@@ -7,3 +7,24 @@ const (
 	TWI_FREQ_100KHZ = 100000
 	TWI_FREQ_400KHZ = 400000
 )
+
+// WriteRegister transmits first the register and then the data to the
```

Comment:
I'd recommend to mention that this is aimed only at 8 bit devices.

Reply:
Good point. I've added that this is only for devices with 7-bit addresses (assuming that is what you meant?)

Reply:
No, I meant that writing a 8 bits "address" implies a 8 bits address space. There are some (like the lepton) with a 16 bits address space.
https://github.com/google/periph/blob/master/devices/lepton/cci/cci.go#L134

It's not a big deal.